group5HM.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('ut2019', function() {
+  	const email = 'test.user@ut2019.ee';
+	const name = 'Test User';
+	const message = 'Custom message!';
+	const urls = ['https://ut2019.voog.com/contact', 'https://ut2019.voog.com/contact-1'];
```

Comment:
You could use the base url https://ut2019.voog.com as a configurable variable and not duplicate it across the tests. In a real-world scenario you most probably will run these tests against different environments and it will help you to easily re-configure your tests against dev/test environments.